### PR TITLE
oudy alusef

### PR DESCRIPTION
**Pull request by OudyUsef19**
H.W2

### CODE REVIEW

File: hw2.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+#Q3
+print(list.count(0)) #print 2
+
+#Q4
+#print(list[4]) #list index out of range
+
+#Q5
+print(2 in list) #True
+
+#Q6
```
Comment (A-Fakher) [inline hw2.py:24]: you need just to append 'A', no need for pop()

File: hw2.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+list.append('A')
+print(list)
+list.pop(4)
+
+#Q7
+list.sort()
+print(list)
+list = [2, 0 ,1 , 0]
+
+#Q8
```
Comment (A-Fakher) [inline hw2.py:34]: you should have used list.pop() instead of  list[0:3]

File: hw2.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+print(list[0:3])
+
+#Q10
```
Comment (A-Fakher) [inline hw2.py:37]: list[1:3] is the answer

File: hw2.py (hunk 1 above)
Comment (OudyUsef19) [reply hw2.py:24]: i need pop it because i need use same list another code without 'A'

**Pull request closed without merging**